packages/core/test/unit/query-generator/table-exists-query.test.ts

```diff
@@ -0,0 +1,74 @@
+      ibmi: `SELECT TABLE_NAME, TABLE_SCHEMA FROM QSYS2.SYSTABLES WHERE TABLE_NAME = 'myTable' AND TABLE_SCHEMA = CURRENT SCHEMA`,
+      mssql: `SELECT TABLE_NAME, TABLE_SCHEMA FROM INFORMATION_SCHEMA.TABLES WHERE TABLE_TYPE = 'BASE TABLE' AND TABLE_NAME = N'myTable' AND TABLE_SCHEMA = N'${defaultSchema}'`,
+      sqlite: `SELECT name FROM sqlite_master WHERE type='table' AND name='myTable'`,
+    });
+  });
+
+  it('produces a table exists query for a model', () => {
+    const MyModel = sequelize.define('MyModel', {});
+
+    expectsql(() => queryGenerator.tableExistsQuery(MyModel), {
+      default: `SELECT TABLE_NAME, TABLE_SCHEMA FROM INFORMATION_SCHEMA.TABLES WHERE TABLE_TYPE = 'BASE TABLE' AND TABLE_NAME = 'MyModels' AND TABLE_SCHEMA = '${defaultSchema}'`,
+      db2: `SELECT TABNAME, TABSCHEMA FROM SYSCAT.TABLES WHERE TABNAME = 'MyModels' AND TABSCHEMA = '${defaultSchema}'`,
+      ibmi: `SELECT TABLE_NAME, TABLE_SCHEMA FROM QSYS2.SYSTABLES WHERE TABLE_NAME = 'MyModels' AND TABLE_SCHEMA = CURRENT SCHEMA`,
+      mssql: `SELECT TABLE_NAME, TABLE_SCHEMA FROM INFORMATION_SCHEMA.TABLES WHERE TABLE_TYPE = 'BASE TABLE' AND TABLE_NAME = N'MyModels' AND TABLE_SCHEMA = N'${defaultSchema}'`,
+      sqlite: `SELECT name FROM sqlite_master WHERE type='table' AND name='MyModels'`,
+    });
+  });
+
+  it('produces a table exists query for a table and schema', () => {
+    expectsql(() => queryGenerator.tableExistsQuery({ tableName: 'myTable', schema: 'mySchema' }), {
+      default: `SELECT TABLE_NAME, TABLE_SCHEMA FROM INFORMATION_SCHEMA.TABLES WHERE TABLE_TYPE = 'BASE TABLE' AND TABLE_NAME = 'myTable' AND TABLE_SCHEMA = 'mySchema'`,
+      db2: `SELECT TABNAME, TABSCHEMA FROM SYSCAT.TABLES WHERE TABNAME = 'myTable' AND TABSCHEMA = 'mySchema'`,
+      ibmi: `SELECT TABLE_NAME, TABLE_SCHEMA FROM QSYS2.SYSTABLES WHERE TABLE_NAME = 'myTable' AND TABLE_SCHEMA = 'mySchema'`,
+      mssql: `SELECT TABLE_NAME, TABLE_SCHEMA FROM INFORMATION_SCHEMA.TABLES WHERE TABLE_TYPE = 'BASE TABLE' AND TABLE_NAME = N'myTable' AND TABLE_SCHEMA = N'mySchema'`,
+      sqlite: `SELECT name FROM sqlite_master WHERE type='table' AND name='mySchema.myTable'`,
+    });
+  });
+
+  it('produces a table exists query for a table and default schema', () => {
+    expectsql(() => queryGenerator.tableExistsQuery({ tableName: 'myTable', schema: dialect.getDefaultSchema() }), {
+      default: `SELECT TABLE_NAME, TABLE_SCHEMA FROM INFORMATION_SCHEMA.TABLES WHERE TABLE_TYPE = 'BASE TABLE' AND TABLE_NAME = 'myTable' AND TABLE_SCHEMA = '${defaultSchema}'`,
+      db2: `SELECT TABNAME, TABSCHEMA FROM SYSCAT.TABLES WHERE TABNAME = 'myTable' AND TABSCHEMA = '${defaultSchema}'`,
+      ibmi: `SELECT TABLE_NAME, TABLE_SCHEMA FROM QSYS2.SYSTABLES WHERE TABLE_NAME = 'myTable' AND TABLE_SCHEMA = CURRENT SCHEMA`,
+      mssql: `SELECT TABLE_NAME, TABLE_SCHEMA FROM INFORMATION_SCHEMA.TABLES WHERE TABLE_TYPE = 'BASE TABLE' AND TABLE_NAME = N'myTable' AND TABLE_SCHEMA = N'dbo'`,
```

Comment:
```suggestion
      mssql: `SELECT TABLE_NAME, TABLE_SCHEMA FROM INFORMATION_SCHEMA.TABLES WHERE TABLE_TYPE = 'BASE TABLE' AND TABLE_NAME = N'myTable' AND TABLE_SCHEMA = N'{defaultSchema}'`,
```
For consistency with the other tests in this file

Reply:
Updated now.